src/horizontal_axis.rs

```diff
@@ -50,7 +50,10 @@ impl PartialEq for Props {
             && self.y1 == other.y1
             && self.tick_len == other.tick_len
             && self.title == other.title
-            && Rc::ptr_eq(&self.scale, &other.scale)
+            && std::ptr::eq(
+                &*self.scale as *const dyn AxisScale as *const u8,
+                &*other.scale as *const dyn AxisScale as *const u8,
+            )
```

Comment:
Can we please document this in reference to https://github.com/rust-lang/rust/issues/46139? It seems like a valid comparison to me as we are comparing allocations.

Reply:
Actually, can be written:

```rust
&*self.scale as *const _ *const u8
```